ramanujan/pcf/pcf_test.py

```diff
@@ -87,3 +88,29 @@ def test_precision_e():
 def test_precision_phi():
     pcf = PCF(1, 1)
     assert PCF.precision(pcf.walk(2**10)) == 427
+
+
+def test_delta_sequence_agrees_with_delta():
+    pcf = PCF(2*n+1, n**2)
+    limit = 4/mp.pi
+    depth = 50
+    
+    test_deltas = pcf.delta_sequence(depth, limit)
```

Comment:
Usually you want to name these `actual_values` (the ones you're testing the functionality of)